job-server/src/spark.jobserver/io/JobFileDAO.scala

```diff
@@ -1,200 +0,0 @@
-package spark.jobserver.io
-
```

Comment:
See my comments in the issue.  Let's not delete it quite yet but mark it as deprecated, with warnings in the config.